example/sparse/linear_classification.py

```diff
@@ -78,18 +78,18 @@ def next(self):
 
 datasets = { 'kdda' : kdda, 'avazu' : avazu }
 
-def regression_model(feature_dim):
-     initializer = mx.initializer.Normal()
+def linear_model(feature_dim):
      x = mx.symbol.Variable("data", stype='csr')
      norm_init = mx.initializer.Normal(sigma=0.01)
-     v = mx.symbol.Variable("v", shape=(feature_dim, 1), init=norm_init, stype='row_sparse')
-     embed = mx.symbol.dot(x, v)
+     weight = mx.symbol.Variable("weight", shape=(feature_dim, 1), init=norm_init, stype='row_sparse')
```

Comment:
Is `weight` a dense vector of size `feature_dim`? In industrial circumstance, the feature id may be generated by hash, so `feature_dim` could be `numeric_limits<uint64_t>`.